js/bitget.js

```diff
@@ -1076,33 +1076,33 @@ module.exports = class bitget extends Exchange {
         if ((symbol === undefined) && (market !== undefined)) {
             symbol = market['symbol'];
         }
-        const last = this.safeNumber2 (ticker, 'last', 'close');
-        const open = this.safeNumber (ticker, 'open');
+        const last = this.safeString2 (ticker, 'last', 'close');
+        const open = this.safeString (ticker, 'open');
         let bidVolume = undefined;
         let askVolume = undefined;
-        let bid = this.safeValue (ticker, 'bid');
+        let bid = this.safeString (ticker, 'bid');
         if (bid === undefined) {
-            bid = this.safeNumber (ticker, 'best_bid');
+            bid = this.safeString (ticker, 'best_bid');
         } else {
-            bidVolume = this.safeNumber (bid, 1);
-            bid = this.safeNumber (bid, 0);
+            bidVolume = this.safeString (bid, 1);
+            bid = this.safeString (bid, 0);
         }
-        let ask = this.safeValue (ticker, 'ask');
+        let ask = this.safeString (ticker, 'ask');
```

Comment:
This is not really correct, ask must remain an object, not a string.

Reply:
The same is true for bid, of course.

Reply:
You are right, they are array.